projects.md

```diff
@@ -0,0 +1,57 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation, all members may pursue leads as they come up. These may be a client or a grant opportunity. Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
+
+If a member decides to pursue a **lead**, they can reach out for preliminary discussions to see whether there is a potential to work together. If the lead expresses intention to proceed with a concrete project in mind, this becomes a **prospect**. The member now leads this sales effort and should draft an **internal proposal** and present it at a meeting to see whether the co-operative wants to move forward to submit a formal proposal to the prospective client. In this meeting, we should discuss any concerns by other members, and gauge team capacity given project timelines. A sample internal proposal should include:
+
+- Project Title
+- Background
+- Scope & Timeline
+- Outcomes & Opportunities
+- Skillsets
+- Budget
+- Communication
+- Risks
+- Resources
+
+A [template](#internal-proposal-template) is included below. You can add more sections as appropriate for the project. This may include sample code, project roadmap, next steps, etc. You can find examples in our archived proposals folder in our [shared drive](http://link.hypha.coop/drive).
+
+## Client Proposal Drafting and Submissions
+
+Project proposals to be submitted to a potential client or grant committee should be drafted by members who expect to participate in the project with the help of members of the Business Planning WG. The WG has expertise in drafting client and grant project proposals, and managing the proposal process, so it is helpful to involve members of that group from the drafting stage. All that are involved in the proposal drafting make up the **proposal team** that will eventually sign off on the final draft to be submitted.
+
+If a project is accepted, this team will lead the kick-off meetings with the client (e.g. discuss terms of payment and project logistics) and gradually transfer the relationship entirely to the **project team**, which may not necessarily include everyone in the proposal team.
```

Comment:
```suggestion
If a project is accepted, this team will lead the initial consultation meetings with the client (e.g. discuss terms of payment and project logistics) and gradually transfer the relationship entirely to the **project team**, which may not necessarily include everyone in the proposal team.
```

Reply:
If others feel a better term please go ahead and merge their preferred term. I don't really care what to call this except it has a name.

Reply:
Yeah, this is prob stuff we can sort in 5 seconds a bizdev meeting convo, not an issue. Next time I'll just flag and queue it :raised_hands: